env/container/redis/Dockerfile

```diff
@@ -0,0 +1,2 @@
+RUN useradd -ms /bin/bash user
```

Comment:
https://github.com/docker-library/redis/blob/master/5.0/Dockerfile#L4

This one is also done for us